externals/kyuubi-spark-sql-engine/src/test/scala/org/apache/kyuubi/engine/spark/SchedulerPoolSuite.scala

```diff
@@ -101,6 +101,10 @@ class SchedulerPoolSuite extends WithSparkSQLEngine with HiveJDBCTestHelper {
       })
       threads.shutdown()
       threads.awaitTermination(20, SECONDS)
+      // make sure the SparkListener has received the finished events for job1 and job2.
```

Comment:
if the root cause is the event not being received, let's call `KyuubiSparkContextHelper.waitListenerBus()` to make sure there are no more events in the spark event queue.

Reply:
Thank you for pointing out, KyuubiSparkContextHelper.waitListenerBus is indeed a better idea.The average execution time is reduced by 1 second.

total_time: 882 ,avg_time: 17
total: 50, success: 50, fail: 0